docs/writing-rules/private-rules.md

```diff
@@ -13,40 +13,46 @@ As we continue to develop and refine this feature, we welcome and appreciate all
 
 ## Getting started
 
-***DEPLOYMENT_ID*** can be found in Semgrep App [here](https://semgrep.dev/manage/settings)
+Run 
 
-***SEMGREP_TOKEN*** can be generated in Semgrep App [here](https://semgrep.dev/manage/settings/tokens)
-
-### Dockerhub
-In a directory with the rule file RULE_YAML you want to upload run:
 ```
-docker pull returntocorp/semgrep-upload:latest
-docker run -v $(pwd):/rules -e SEMGREP_UPLOAD_DEPLOYMENT=DEPLOYMENT_ID -e SEMGREP_TOKEN=SOME_TOKEN returntocorp/semgrep-upload:latest /rules/RULE_YAML
+semgrep login
+semgrep publish myrules/
```

Comment:
You cannot put two separate actions into one code snippet.

Where do we need to run this command (the original document states that you need to run it in a directory where you have your YAML rule file).

1. Run the following command (TODO - evaluate - in the file where you have YAML rule file):
```
semgrep login
```

It seems that there are more steps that this procedure missed.

![Screenshot 2022-02-25 at 15 01 40](https://user-images.githubusercontent.com/26669307/155733204-d72ea656-8b45-4481-b599-1f0a72e3c7a5.jpg)

2.  Run the following command:
```
semgrep publish myrules/
```